libtransmission/rpc-server.cc

```diff
@@ -77,39 +77,101 @@ auto inline constexpr TrUnixAddrStrLen = size_t{ sizeof(((struct sockaddr_un*)nu
 
 enum tr_rpc_address_type
 {
-    TR_RPC_AF_INET,
-    TR_RPC_AF_INET6,
-    TR_RPC_AF_UNIX
+    TR_RPC_UNIX_ADDR,
+    TR_RPC_INET_ADDR,
+    TR_RPC_NUM_ADDR
```

Comment:
Doesn't look like `TR_RPC_NUM_ADDR` is used. Do we need it?

Reply:
I was trying to mirror what `tr_address` does, but no, it is not necessary